catroid/src/main/java/org/catrobat/catroid/ui/ProjectActivity.java

```diff
@@ -192,40 +190,18 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 		}
 	}
 
-	private boolean needToShowLegoEV3InfoDialog() {
-		boolean isLegoEV3InfoDialogDisabled = SettingsFragment
-				.getShowLegoEV3MindstormsSensorInfoDialog(getApplicationContext());
+	private void showLegoSensorConfigInfo() {
+		int requiredResources = ProjectManager.getInstance().getCurrentProject().getRequiredResources();
+		boolean nxtDialogEnabled = SettingsFragment.getLegoNXTSensorInfoDialogEnabled(this);
```

Comment:
remove getLegoNXTSensorInfoDialogEnabled from SettingsFragment and get directly from sharedprefs

Reply:
done - note: some settings regarding lego are still in the SettingsFragment. Removing that probably should be a separate ticket